src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpoint.java

```diff
@@ -207,7 +208,7 @@ public KeyMaterialFactory newKeyMaterialFactory(Item context, @Nonnull VirtualCh
     }
 
     /**
-     * @deprecated Call {@link #newKeyMaterialFactory(Item, FilePath, Launcher, TaskListener, String)}
+     * @deprecated Call {@link #newKeyMaterialFactory(Item, FilePath, Launcher, EnvVars, TaskListener, String)}
```

Comment:
nit: Maybe also a good idea to update the recommended method in the Javadoc of the two deprecated methods above this one.

Reply:
Yes, good catch.